benchmarks/models.py

```diff
@@ -155,11 +155,16 @@ class Status:
         SUCCESS = 'successful'
         FAILURE = 'failure'
 
+    # ID now auto-increments
+    id = models.AutoField(primary_key=True)
```

Comment:
I believe we can just leave this out, django will automatically create
![image](https://user-images.githubusercontent.com/5308236/224102159-15b9e880-2dd4-4414-9064-1313ddccffdb.png)